source/jormungandr/tests/line_sections_tests.py

```diff
@@ -312,11 +518,12 @@ def test_journeys_use_vj_impacted_by_line_section(self):
         In this test we leave at 08:00 so we'll use vj:1:1 (that is impacted) during the application
         period of the impact
         """
-        date = '20170102T080000'  # with this departure time we should take 'vj:1:1' that is impacted
+        # with this departure time we should take 'vj:1:1' that is impacted
+        date = '20170102T080000'
         current = '_current_datetime=20170101T080000'
 
         def journey_query(_from, to):
-            return 'journeys?from={fr}&to={to}&datetime={dt}&{cur}&data_freshness=base_schedule'.format(
+            return 'journeys?from={fr}&to={to}&datetime={dt}&{cur}' '&data_freshness=base_schedule'.format(
```

Comment:
I just saw this, I think black rewrited 2 lines in one and made this :3 I would like to change this before a merge.